release_scripts/Packages.Microsoft/publish.sh

```diff
@@ -0,0 +1,100 @@
+# Publishes deb and rpm packages to Packages.Microsoft repo
```

Comment:
I'm curious why we didn't do this in python?

Reply:
Looking back in hindsight... we could have. Pros and cons here. I thought the bash script would have been more elegant because I pictured fewer complications than our result.

What's nice here is you don't need any Python, it works in any container with bash so this script can theoretically be used in any Linux container (using arm). But Python would have been a cleaner looking script.